modules/ROOT/pages/edifact/edifact-edi-connector.adoc

```diff
@@ -5,9 +5,13 @@ endif::[]
 
 Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium]
 
+EDIFACT EDI Connector v2.0
+
 The EDIFACT EDI connector lets you convert EDIFACT messages to and from
 DataWeave-compatible representations using lists and maps. This connector supports EDIFACT versions D.93A, D.95B, D.96A, D.97A, D.98B, D.99A, D.99B, D.00A, D.01B, D.02A, D.03A with supplied schema definitions for all standard messages.
 
+Exchange: https://www.mulesoft.com/exchange/com.mulesoft.connectors/mule-edifact-extension/[EDIFACT EDI Connector]
```

Comment:
All of these Exchange links are being moved to the See Also section at the bottom of the page. They were intentionally removed from the top of the file.